website/docs/blog.md

```diff
@@ -3,7 +3,112 @@ id: blog
 title: Blog
 ---
 
-_This section is a work in progress. [Welcoming PRs](https://github.com/facebook/docusaurus/issues/1640)._
+## Initial Setup
+
+To setup your site's blog, start by creating a `blog` directory within your repo's `my-website` directory.
+
+Then, add a navbar link to your blog within `docusaurus.config.js`:
+
+```js
+links: [
+    ...
+    {to: 'blog', label: 'Blog', position: 'left'}, // position: 'left' for creating Blog link on the left side of navbar, position: 'right' for creating Blog link on the right side of navbar
+    ...
+]
+```
+
+## Adding Posts
+
+To publish in the blog, create a file within the blog directory with a formatted name of `YYYY-MM-DD-my-blog-post-title.md`. The post date is extracted from the file name.
```

Comment:
shall we mention that the blog directory is by default `blog/` under the project root, and add an "options" section at the end of this doc that goes through the customizable options?

Reply:
Yes. That would be better I think. Would make the required changes.